core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -1254,7 +1255,7 @@ class SparkSubmitSuite
         |public void init(PluginContext ctx, Map<String, String> extraConf) {
         |  String str = null;
         |  try (BufferedReader reader =
-        |    new BufferedReader(new InputStreamReader(new FileInputStream($tempFileName)))) {
+        |    new BufferedReader(new InputStreamReader(new FileInputStream("$tempFileName")))) {
```

Comment:
Oh, interesting. Previously, this test case succeeds with `"`?

Reply:
Actually, I replaced `"test.txt"` with `$tempFileName` just before first push so, it's a miss-replacement.